phpunit.xml.dist

```diff
@@ -18,6 +18,12 @@
 		<testsuite name="speculation-rules">
 			<directory suffix=".php">./tests/plugins/speculation-rules</directory>
 		</testsuite>
+		<testsuite name="dominant-color-images">
+			<directory suffix=".php">./tests/plugins/dominant-color-images</directory>
+		</testsuite>
+		<testsuite name="webp-uploads">
+			<directory suffix=".php">./tests/plugins/webp-uploads</directory>
+		</testsuite>
```

Comment:
Nit-pick: In e.g. `composer.json` and `plugins.json`, you sort the plugins alphabetically. Maybe we can do that here too?
```suggestion
		<testsuite name="dominant-color-images">
			<directory suffix=".php">./tests/plugins/dominant-color-images</directory>
		</testsuite>
		<testsuite name="speculation-rules">
			<directory suffix=".php">./tests/plugins/speculation-rules</directory>
		</testsuite>
		<testsuite name="webp-uploads">
			<directory suffix=".php">./tests/plugins/webp-uploads</directory>
		</testsuite>
```

Reply:
Seems like I missed it while resolving merge conflicts.